onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -140,4 +140,19 @@ def optimize_for_ort(
     )
     # Apply the ORT pattern rewrite rules.
     rewrite(model, ORT_PATTERN_REWRITE_RULES)
+
+    passes = ir.passes.Sequential(
+        # TODO(exporter team): Fold transpose into initializers
```

Comment:
This should be done already

Reply:
https://github.com/microsoft/onnxscript/blob/2cc25021e10c564bdc479961c2721390b3a943ef/onnxscript/optimizer/_constant_folding.py#L903